.eslintrc

```diff
@@ -1,70 +1,82 @@
 {
   "parser": "@typescript-eslint/parser",
+  "env": {
+    "node": true
```

Comment:
this was added to fix these errors

<img width="752" alt="Screenshot 2022-12-09 at 15 21 55" src="https://user-images.githubusercontent.com/1225343/206723090-a2aae6f3-6873-44ca-9f87-6f5efad93bdd.png">

<img width="857" alt="Screenshot 2022-12-09 at 15 23 04" src="https://user-images.githubusercontent.com/1225343/206723247-f54f4c6c-2c9e-4f43-bcef-806ed6f9d63c.png">